src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -1056,9 +1069,17 @@ void mongoSubCacheStatisticsGet(int* refreshes, int* inserts, int* removes, int*
   {
     while (cSubP != NULL)
     {
+      char msg[256];
+
       unsigned int bytesLeft = listSize - strlen(list);
 
-      if (strlen(cSubP->subscriptionId) + 2 > bytesLeft)
+#if UpdateBugTests
+      snprintf(msg, sizeof(msg), "%s|N:%lu|E:%lu|T:%lu|P:%d", cSubP->subscriptionId, cSubP->lastNotificationTime, cSubP->expirationTime, cSubP->throttling, cSubP->pendingNotifications);
+#else
+      snprintf(msg, sizeof(msg), "%s", cSubP->subscriptionId);
+#endif
+
+      if (strlen(msg) + 2 > bytesLeft)
```

Comment:
Sure, why not. It's to make sure ", " has room in the char-vector - but that you knew already